google/resumable_media/requests/_helpers.py

```diff
@@ -33,6 +34,12 @@
 # The number of seconds to wait between bytes sent from the server.
 _DEFAULT_READ_TIMEOUT = 60
 
+_SLOW_CRC32C_WARNING = (
+    "Currently using crcmod in pure python form. This is a slow "
+    "implementation. You may consider installing `google-crc32c` for a "
+    "precompiled wheel with fast crc32c support."
+)
```

Comment:
IIUC this is only possible when Python is 2.x. I would get confused as a user because google-crc32c python package is a requirement for installation.

Reply:
Okay, PTAL